<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统监控与自动化工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="systemMonitorTab">
       <attribute name="title">
        <string>系统监控</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="cpuGroupBox">
          <property name="title">
           <string>CPU监控</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QProgressBar" name="cpuProgressBar">
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cpuPercentLabel">
               <property name="text">
                <string>0%</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>CPU使用率阈值：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="cpuThresholdSpinBox">
               <property name="suffix">
                <string>%</string>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>80</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="memoryGroupBox">
          <property name="title">
           <string>内存监控</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QProgressBar" name="memoryProgressBar">
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="memoryPercentLabel">
               <property name="text">
                <string>0%</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>内存使用率阈值：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="memoryThresholdSpinBox">
               <property name="suffix">
                <string>%</string>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>80</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fileManagerTab">
       <attribute name="title">
        <string>文件管理</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="notificationTab">
       <attribute name="title">
        <string>通知设置</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="reportTab">
       <attribute name="title">
        <string>报告生成</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui>